data/lang/module-findperson/en.json

```diff
@@ -0,0 +1,550 @@
+    "description": "",
+    "message": "Hello, my name is {client}. {wanted} didn't return from a holiday trip to his workplace at the {company}. He was last seen in the {system} ({sectorx}, {sectory}, {sectorz}) system. {wanted} has knowledge of some vital business secrets. So we need someone who can get a few answers from him about his ongoing absence. We pay {cash} for this mission. {system} is in a distance of {dist} ly."
+  },
+  "INTROTEXT_TRAITOR_MALE_2": {
+    "description": "",
+    "message": "Good day. The {company} is searching for an employee who stole some classified documents. His name is {wanted}. Management calls him a traitor, but wants to give him the opportunity to reconsider and repent. We pay {cash} if you can find him in the {system} ({sectorx}, {sectory}, {sectorz}) system, {dist} ly distance, and deliver a message to him."
+  },
+  "IS_THERE_A_RISK": {
+    "description": "",
+    "message": "Is there any risk?"
+  },
+  "MERCENARY_GREETING_FEMALE_1": {
+    "description": "",
+    "message": "I'm sorry. Your death is my business!"
+  },
+  "MERCENARY_GREETING_FEMALE_2": {
+    "description": "",
+    "message": "I'll cut you to slices!"
+  },
+  "MERCENARY_GREETING_FEMALE_3": {
+    "description": "",
+    "message": "You want to meet {wanted}? I will make you meet your maker first!"
+  },
+  "MERCENARY_GREETING_MALE_1": {
+    "description": "",
+    "message": "It's nothing personal. I kill you for a fee!"
+  },
+  "MERCENARY_GREETING_MALE_2": {
+    "description": "",
+    "message": "I'll send you to hell!"
+  },
+  "MERCENARY_GREETING_MALE_3": {
+    "description": "",
+    "message": "You want to meet {wanted}? Over my dead body!"
+  },
+  "OK_AGREED": {
+    "description": "",
+    "message": "OK, agreed."
+  },
+  "PIRATE_ANSWER_1": {
+    "description": "",
+    "message": "I have transferred an answer for {client} to you. Good bye."
+  },
+  "PIRATE_ANSWER_2": {
+    "description": "",
+    "message": "Please pass the message I'm currently transferring to you to {client}. Bye."
+  },
+  "PIRATE_ANSWER_3": {
+    "description": "",
+    "message": "Haha, Please tell this {client} I've nothing to say. Have a nice day."
+  },
+  "PIRATE_GREETING_1": {
+    "description": "",
+    "message": "I heard you were looking for me. You have my attention. What do you want?"
+  },
+  "PIRATE_GREETING_2": {
+    "description": "",
+    "message": "OK, you got me. Don't waste time and tell me what you want!"
+  },
+  "PIRATE_GREETING_3": {
+    "description": "",
+    "message": "I heard you were supposed to give me a message?"
+  },
+  "PIRATE_TAUNTS_1": {
+    "description": "",
+    "message": "You want an answer? Well, here it comes!"
+  },
+  "PIRATE_TAUNTS_2": {
+    "description": "",
+    "message": "Enough of this! Prepare to go to hell!"
+  },
+  "PIRATE_TAUNTS_3": {
+    "description": "",
+    "message": "You actually went all this way to deliver this stupid message to me? You better get out of here!"
```

Comment:
I'd prefer "silly" over "stupid", a word/insult we've not used thus far in the game. "Silly" feels less harsh, more child friendly.